src/main/java/seedu/address/model/candidate/Availability.java

```diff
@@ -3,34 +3,43 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
 /**
  * Represents a Candidate's availability.
- * Guarantees: immutable; is valid as declared in {@link #isValidDate(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidDay(String)}
  */
 public class Availability {
 
-    public static final String MESSAGE_CONSTRAINTS = "Availability should consists of a list of numbers from 1 to 7, "
-            + "separated by commas ','";
-    public static final String VALIDATION_REGEX = "^[1-7]{1}+(?:,[1-7]{1}+)*$";
+    public static final String MESSAGE_CONSTRAINTS = "Availability should consists of a list of 5 numbers, "
+            + "ranging from 1 to 5, separated by commas ','\nEg. avail/1,2,3,4,5";
+    public static final String VALIDATION_REGEX = "^[1-5]{1}+(?:,[1-5]{1}+)*$";
+    private static final String[] WEEK = {"Mon", "Tue", "Wed", "Thu", "Fri"};
 
     public final String availability;
-
+    private List<String> availabilityList = new ArrayList<>();
     /**
      * Constructs a {@code Availability}.
      *
      * @param availability A valid available date.
```

Comment:
Quite a small point but the JavaDoc could also be updated from `date` to `day`, leaving it here first in case we forget to catch it next time during bug checking 🥲